pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -107,6 +108,32 @@ func TestGenCerts(t *testing.T) {
 		t.Fatal(err)
 	}
 	if err := GenCerts(TestCertsTmp, TestCaCertPath, TestCaKeyPath, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
-		fmt.Println(err)
+		t.Fatal(err)
+	}
+
```

Comment:
we call GenCerts twice by different internal implementation, since you want to checkCertFiles, it is best to test fully and test both.

Reply:
I have modified this code in the latest rebase, please check it again

Reply:
thanks~ But this is not perfect enough, supposing first `GenCerts` success and second `GenCerts` fails, since the certs generated by first `GenCerts` still exist, it can be detected by second `checkCertFiles`.

maybe we can set the output dir of second `GenCerts` to other path?